lib/p2p/NodeList.ts

```diff
@@ -12,6 +12,9 @@ const reputationEventWeight = {
   [ReputationEvent.PacketTimeout]: -1,
   [ReputationEvent.SwapFailure]: -10,
   [ReputationEvent.SwapSuccess]: 1,
+  [ReputationEvent.InvalidMessage]: -1,
+  [ReputationEvent.UnknownMessageType]: -1,
+  [ReputationEvent.UnparsableMessage]: -1,
```

Comment:
I think the old scores made a little more sense (10, 10, and 20 for `UnknownMessageType`). I'd probably go back to those, or maybe make it 5, 5, and 20, because the unknown message type implies a deliberately modified version of the software.

Reply:
Yeah, I wasn't sure about the penalty value. I'll go back to the original values.